src/components/Compare/CompareSelector.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Grid } from "@trussworks/react-uswds";
+import { Dispatch, SetStateAction } from "react";
+import { CompareProviders } from "../../pages/Compare";
+import Selections from "./Selections";
+
+type CompareSelectorProps = {
+  providers: CompareProviders;
```

Comment:
i think this prop should be a list instead of explicitly having provider A & B, b/c it feels slightly more intuitive if the providers show up in the order they are selected. so, when clearing the first provider, the second provider moves into the first slot, instead of having fixed slots for the providers